airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml

```diff
@@ -87,6 +87,13 @@ data:
               - "creatives"
               - "campaign_groups"
               - "conversions"
+      5.0.0:
```

Comment:
@lazebnyi can we separate the change to the primary key to avoid the breaking change?

IMO we should always strive to release in the smallest increment possible to reduce the risk of borking a release. The change in primary key is important, but does not have to be coupled with the migration to low-code, which is a major refactor.

Reply:
breaking changes moved to separate PR - https://github.com/airbytehq/airbyte/pull/45353